pymc/step_methods/metropolis.py

```diff
@@ -308,16 +308,18 @@ def astep(self, q0: RaveledVars) -> tuple[RaveledVars, StatsType]:
         else:
             accept_rate = self.delta_logp(q, q0d)
             q, accepted = metrop_select(accept_rate, q, q0d, rng=self.rng)
-            self.accept_rate_iter = accept_rate
+            self.accept_rate_iter[:] = accept_rate
```

Comment:
Nitpick, looks like the line below
```suggestion
            self.accept_rate_iter[0] = accept_rate
```